caselaw/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
+Django settings for caselaw project.
+
+Generated by 'django-admin startproject' using Django 3.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+import django_heroku
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'o#-@3p*_2*#jf=cut@f5@3yy*m!*loiuc)7r)%w+nec5dg+ne@'
```

Comment:
This is one of the reasons we shouldn't be committing it. I assume we'll use separate settings for production, so it's fine if these are just local settings.

Reply:
Can we keep the file if we switch it to use environment variables?

Reply:
Yes, [environment variables](https://devcenter.heroku.com/articles/getting-started-with-python?singlepage=true#define-config-vars) are configurable in Heroku too. I added a link in line 125.